html/rendering/non-replaced-elements/the-fieldset-element-0/legend-block-formatting-context.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<title>The legend element: block formatting context</title>
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<style>
+/* Set margin and padding for fieldset to 0 to make things simpler */
+fieldset {
+  margin: 0;
+  padding: 0;
+}
+.wrapper {
+  height: 200px;
+  position: relative;
+}
+.float {
+  float: left;
+  width: 50px;
+  height: 50px;
+  background-color: orange;
+}
+</style>
+
+<div class=wrapper>
+ <div class=float></div>
+ <legend id=in-body><div class=float></div></legend>
+ <div class=float></div>
+</div>
+
+<div class=wrapper>
+ <fieldset>
+  <div class=float></div>
```

Comment:
Yes. https://html.spec.whatwg.org/#rendered-legend says what is the rendered legend. What you cite is an authoring requirement.